prime-dotnet-webapi/Services/Rules/AutomaticAdjudicationRules.cs

```diff
@@ -99,39 +100,70 @@ record = await _collegeLicenceClient.GetCollegeRecordAsync(cert.Prefix, cert.Lic
                 catch (PharmanetCollegeApiException)
                 {
                     enrollee.AddReasonToCurrentStatus(StatusReasonType.PharmanetError, cert.ToString());
-                    await _businessEventService.CreatePharmanetApiCallEventAsync(enrollee.Id, cert.Prefix, cert.LicenseNumber, "An error occurred calling the Pharmanet API.");
+                    await _businessEventService.CreatePharmanetApiCallEventAsync(enrollee.Id, cert.Prefix, cert.LicenseNumber, "An error occurred calling the PharmaNet API.");
                     passed = false;
                     continue;
                 }
 
                 //After validating with prescriber prefix and no hit, try to validate with non-prescriber prefix
                 if (record == null)
                 {
-                    await _businessEventService.CreatePharmanetApiCallEventAsync(enrollee.Id, cert.Prefix, cert.LicenseNumber, "Record not found in Pharmanet.");
-                    if (cert.NonPrescribingPrefix != null)
+                    await _businessEventService.CreatePharmanetApiCallEventAsync(enrollee.Id, cert.Prefix, cert.LicenseNumber, "Record not found in PharmaNet.");
+                }
+                else
+                {
+                    await _businessEventService.CreatePharmanetApiCallEventAsync(enrollee.Id, cert.Prefix, cert.LicenseNumber,
+                        $"A record was found in PharmaNet with effective date {record.EffectiveDate:dd MMM yyy} and status {record.Status}.");
```

Comment:
Minor thing, can the date format be `d MMM yyyy` (two occurrences) 

![image](https://github.com/bcgov/moh-prime/assets/7586665/4fa836f9-7fc4-4e42-887d-79825442cd55)

Reply:
fixed.